File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        String str = "0123456789";
        
        for(int i = 0; i<numbers.length; i++){
            if(str.contains(Integer.toString(numbers[i]))) answer += numbers[i];
        }
        return 45 - answer;
    }
}

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.js
function solution(numbers) {
    var answer = 0;
    var str = "0123456789";
    
    for(let i = 0; i<numbers.length; i++){
        if(str.includes(String(numbers[i]))) answer += numbers[i];
    }
    return 45- answer;
}